sdks/python/apache_beam/transforms/combiners_test.py

```diff
@@ -482,9 +492,20 @@ def test_combining_with_accumulation_mode_and_fanout(self):
               trigger=AfterWatermark(early=AfterAll(AfterCount(1))))
           | beam.CombineGlobally(sum).without_defaults().with_fanout(2))
 
-      # The frings for DISCARDING mode is [1, 2, 3, 4, 5, 0, 0].
-      firings = [1, 3, 6, 10, 15, 15, 15]
-      assert_that(result, equal_to(firings))
+      # Partition the result into early_firings and _.
+      # In ACCUMULATING mode, the early_frings is [1, 3, 6, 10],
+      # other_firings is [15, 15, ...]. Different runners have different
+      # number of 15s, but there should be at least one 15.
+      smaller_than_fifteen, fifteen, greater_than_fifteen = (
```

Comment:
Rather than do a partition, I would write something like

```
def has_expected_values(actual):
  ordered = sorted(actual)
  # Early firings.
  hamcrest.assert_that(ordered[:4], hamcrest.contains(1, 3, 6, 10))
  # Different runners have different number of 15s, but there should be at least one 15.
  hamcrest.assert_that(set(ordered[4:]), hamcrest.contains(15))
assert_that(result, has_expected_values)
```